Error: Library "http" not found.

I get this error very often with the new master:
```
opam install http cohttp

<><> Synchronising pinned packages ><><><><><><><><><><><><><><><><><><><><><><>
[http.6.0.0] synchronised (no changes)
[cohttp.6.0.0] synchronised (no changes)

The following actions will be performed:
  - install http   6.0.0*
  - install cohttp 6.0.0*
===== 2 to install =====

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
-> retrieved cohttp.6.0.0  (no changes)
-> retrieved http.6.0.0  (no changes)
-> installed http.6.0.0
[ERROR] The compilation of cohttp.6.0.0 failed at "dune build -p cohttp -j 11
        --promote-install-files=false @install".

#=== ERROR while compiling cohttp.6.0.0 =======================================#
# context     2.1.0 | macos/x86_64 | ocaml.4.13.1 | pinned(git+file:///Users/mseri/code/ocaml/ocaml-cohttp#master#10b87dc8)
# path        ~/.opam/ocaml.4.13.1/.opam-switch/build/cohttp.6.0.0
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -p cohttp -j 11 --promote-install-files=false @install
# exit-code   1
# env-file    ~/.opam/log/cohttp-59307-683628.env
# output-file ~/.opam/log/cohttp-59307-683628.out
### output ###
# File "cohttp/src/dune", line 13, characters 13-17:
# 13 |   (re_export http)
#                   ^^^^
# Error: Library "http" not found.
# Hint: try:
#   dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules --default-target @install --always-show-command-line --promote-install-files false --promote-install-files --release --only-packages cohttp -p cohttp --profile release -j 11 @install
```

--- a/cohttp-async.opam
+++ b/cohttp-async.opam
@@ -26,6 +26,7 @@ bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "dune" {>= "2.9"}
   "ocaml" {>= "4.08"}
+  "http" {= version}
   "cohttp" {= version}
   "async_kernel" {>= "v0.14.0"}
   "async_unix" {>= "v0.14.0"}

--- a/cohttp-lwt-jsoo.opam
+++ b/cohttp-lwt-jsoo.opam
@@ -25,6 +25,7 @@ bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "dune" {>= "2.9"}
   "ocaml" {>= "4.08"}
+  "http" {= version}
   "cohttp" {= version}
   "cohttp-lwt" {= version}
   "logs"

--- a/cohttp-lwt-unix.opam
+++ b/cohttp-lwt-unix.opam
@@ -29,6 +29,7 @@ bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "dune" {>= "2.9"}
   "ocaml" {>= "4.08"}
+  "http" {= version}
   "cohttp" {= version}
   "cohttp-lwt" {= version}
   "cmdliner"

--- a/cohttp-lwt.opam
+++ b/cohttp-lwt.opam
@@ -27,6 +27,7 @@ bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "dune" {>= "2.9"}
   "ocaml" {>= "4.08"}
+  "http" {= version}
   "cohttp" {= version}
   "lwt" {>= "2.5.0"}
   "sexplib0"

--- a/cohttp.opam
+++ b/cohttp.opam
@@ -35,6 +35,7 @@ doc: "https://mirage.github.io/ocaml-cohttp/"
 bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "dune" {>= "2.9"}
+  "http" {= version}
   "ocaml" {>= "4.08"}
   "re" {>= "1.9.0"}
   "uri" {>= "2.0.0"}

--- a/dune-project
+++ b/dune-project
@@ -38,6 +38,7 @@ debugging of the connection layer can be obtained by setting `CONDUIT_DEBUG`
 to any value.
 ")
  (depends
+  (http (= :version))
   (ocaml (>= 4.08))
   (re (>= 1.9.0))
   (uri (>= 2.0.0))
@@ -75,6 +76,7 @@ version of the library. All of these implementations share the same IO logic
 from this module.")
  (depends
   (ocaml (>= 4.08))
+  (http (= :version))
   (cohttp (= :version))
   (lwt (>= 2.5.0))
   sexplib0
@@ -97,6 +99,7 @@ should also be fine under Windows too.
 ")
  (depends
   (ocaml (>= 4.08))
+  (http (= :version))
   (cohttp (= :version))
   (cohttp-lwt (= :version))
   cmdliner
@@ -121,6 +124,7 @@ and also to native JavaScript via js_of_ocaml.
 ")
  (depends
   (ocaml (>= 4.08))
+  (http (= :version))
   (cohttp (= :version))
   (cohttp-lwt (= :version))
   logs
@@ -143,6 +147,7 @@ client and server respectively.
 ")
  (depends
   (ocaml (>= 4.08))
+  (http (= :version))
   (cohttp (= :version))
   (async_kernel (>= v0.14.0))
   (async_unix (>= v0.14.0))

